### PR TITLE
test: added tests for outgoingPayment controller

### PR DESCRIPTION
**Pull request by devcer**
### Context

- fixes https://github.com/interledger/testnet/issues/683

### Changes

Added tests for outgoing payment controller

### CODE REVIEW

File: packages/wallet/backend/tests/outgoingPayment/controller.test.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  MockResponse
+} from 'node-mocks-http'
+import type { AuthService } from '@/auth/service'
+import { applyMiddleware } from '@/tests/utils'
+import { withSession } from '@/middleware/withSession'
+import { mockLogInRequest } from '../mocks'
+import { createUser } from '@/tests/helpers'
+import { OutgoingPaymentController } from '@/outgoingPayment/controller'
+
+describe('outgoingPayment Controller', (): void => {
```
Comment (Tymmmy) [inline packages/wallet/backend/tests/outgoingPayment/controller.test.ts:22]:
```suggestion
describe('OutgoingPayment Controller', (): void => {
```

File: packages/wallet/backend/tests/outgoingPayment/controller.test.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    await truncateTables(knex)
+  })
+
+  describe('create', (): void => {
+    it('should create quote by id', async (): Promise<void> => {
+      const depsMocked = {
+        outgoingPaymentService: {
+          createByQuoteId: () => {}
+        }
+      }
```
Comment (rico191013) [inline packages/wallet/backend/tests/outgoingPayment/controller.test.ts:78]: can we put this depsMocked to tests/mocks.ts file?

File: packages/wallet/backend/tests/outgoingPayment/controller.test.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  afterAll(async (): Promise<void> => {
+    appContainer.stop()
+    knex.destroy()
+  })
+
+  afterEach(async (): Promise<void> => {
+    await truncateTables(knex)
+  })
+
+  describe('create', (): void => {
```
Comment (rico191013) [inline packages/wallet/backend/tests/outgoingPayment/controller.test.ts:72]: I think we're missing tests for failure cases?

File: packages/wallet/backend/tests/outgoingPayment/controller.test.ts (hunk 2 above)
Comment (devcer) [reply packages/wallet/backend/tests/outgoingPayment/controller.test.ts]: done.

File: packages/wallet/backend/tests/outgoingPayment/controller.test.ts (hunk 3 above)
Comment (devcer) [reply packages/wallet/backend/tests/outgoingPayment/controller.test.ts]: added missing tests for failure cases.

**Comment by devcer** [comment]
@rico191013 I have pushed some more changes. let me know if there is any other feedback.

**Review by rico191013 — APPROVED** [review]

**Review by Tymmmy — APPROVED** [review]

**Pull request closed without merging**